test/model/test/model_basic/main.cpp

```diff
@@ -342,12 +344,37 @@ test_model_basic_wt(void)
     wt_model_update_both(table, uri, key1, value1, 80, false);
     wt_model_update_both(table, uri, key1, value1, 85, false);
 
+    /* Verify. */
     testutil_assert(table->verify_noexcept(conn));
 
     /* Now try to get the verification to fail. */
     testutil_check(table->remove(key2, 1000));
     testutil_assert(!table->verify_noexcept(conn));
 
+    /* Close and reopen the database. We must do this for debug log printing to work. */
+    testutil_check(session->close(session, nullptr));
+    testutil_check(conn->close(conn, nullptr));
+    testutil_wiredtiger_open(opts, home, ENV_CONFIG, nullptr, &conn, false, false);
+    testutil_check(conn->open_session(conn, nullptr, nullptr, &session));
+
+    /* Verify using the debug log. */
+    model::kv_database db_from_debug_log;
+    model::debug_log_parser::from_debug_log(db_from_debug_log, conn);
+    testutil_assert(db_from_debug_log.table("table")->verify_noexcept(conn));
+
+    /* Print the debug log to JSON. */
+    std::string tmp_json = create_tmp_file(home, "debug-log-", ".json");
```

Comment:
The temporary file is created in the test's database directory, which gets automatically removed at the end of a successful test run (unless the `-p` command line parameter is specified). I guess it's not really a temporary file then? In which case I could switch the code to just use a regular file name instead of a temporary file name...